rj_gameplay/rj_gameplay/action/pivot.py

```diff
@@ -13,20 +13,37 @@
 
 class Pivot(action.IFiniteAction):
 
-    """
-    Pivot Skill
-    """
-    def __init__(self, robot_id : int, pivot_point: np.ndarray, target_point: np.ndarray, priority : int = 1):
+    def __init__(self, robot_id:int, pivot_point:np.ndarray, target_point:np.ndarray, dribble_speed:float, priority:int=1):
+        self.robot_id = robot_id
         self.pivot_point = pivot_point
         self.target_point = target_point
+        self.dribble_speed = dribble_speed
 
-    def tick(self, intent) -> None:
+    def tick(self, intent: RobotIntent) -> None:
+        new_intent = intent
         pivot_command = PivotMotionCommand()
         pivot_command.pivot_point = Point(x=self.pivot_point[0], y=self.pivot_point[1])
-        intent.motion_command.pivot_motion_command = [pivot_command]
-        intent.active = True
+        pivot_command.pivot_target = Point(x=self.target_point[0], y=self.target_point[1])
+        new_intent.motion_command.pivot_command = [pivot_command]
+        new_intent.dribbler_speed = self.dribble_speed
+        new_intent.is_active = True
+        return new_intent
+
+    def is_done(self, world_state:rc.WorldState) -> bool:
+        #TODO: Change this when we get action state feedback
+        angle_threshold = 0.1
```

Comment:
This is definitely too big, it's about 6°. It should be closer to 1° (0.02 or so)